task_8_9_11.py

```diff
@@ -0,0 +1,42 @@
+from utils import print_intro, validate_number_input
+
+print_intro()
+
+
+def get_valid_number_input(message):
+    number = input(message)
+    while not validate_number_input(number):
+        number = input(message)
+    return int(number)
```

Comment:
you should move it to utils and combine its functionality with validate_number_input.
because now validate_number_input just perform redundant step.
you don't need to return true or false try to return casted value directly